apps/frontend/src/components/GlobalSpinner.tsx

```diff
@@ -0,0 +1,46 @@
+import LogoIcon from "../assets/images/logo-icon.png";
+import Logo from "../assets/images/logo.png";
+import Spinner from "/assets/images/Spinner.png";
+
+type Props = {
+  message?: string;
+};
+
+export const GlobalSpinner = ({ message }: Props) => {
+  return (
+    <div
+      className="flex h-screen w-full flex-col bg-intg-black"
+      style={{
+        backgroundImage:
+          "radial-gradient(rgba(28, 15, 89, 0.30) 50%, rgba(5, 5, 5, 0.30))",
+        backgroundRepeat: "no-repeat",
+        backgroundPosition: "center",
+        backgroundSize: "cover",
+      }}
+    >
+      <style>
+        {`
+         .spinner__circle {
+          overflow: hidden;
```

Comment:
@afolasope  `overflow` and `border-radius` are not really doing anything on the image here, we can also remove the style tag and give the image the tailwind className `animate-spin`.